mne_icalabel/features/topomap.py

```diff
@@ -0,0 +1,121 @@
+import numpy as np
+from mne.channels.layout import _find_topomap_coords
+from mne.defaults import _BORDER_DEFAULT, _EXTRAPOLATE_DEFAULT, _INTERPOLATION_DEFAULT
+from mne.io import Info
+from mne.io.pick import _get_channel_types, _pick_data_channels, pick_info
+from mne.preprocessing import ICA
+from mne.viz.topomap import _check_extrapolate, _make_head_outlines, _setup_interp
+from mne.viz.utils import _setup_vmin_vmax
+
+
+def get_topo_array(ica: ICA, picks="eeg"):
+    """Generate an array of scalp topographies plot of size (n_pixels, n_pixels).
+
+    Parameters
+    ----------
+    ica : ICA
+        Instance of MNE ICA Decomposition mne.preprocesssing.ICA.
+    picks : str | list | slice | None
+        In lists, channel type strings (e.g., ['meg', 'eeg']) will pick channels
+        of those types, channel name strings (e.g., ['MEG0111', 'MEG2623'] will
+        pick the given channels. Can also be the string values “all” to pick all
+        channels, or “data” to pick data channels. None (default) will pick all
+        channels. Note that channels in info['bads'] will be included if their
+        names or indices are explicitly provided.
+
+    Returns
+    -------
+    topo_array : np.ndarray of shape (n_components, n_pixels, n_pixels)
+    """
+    n_components = ica.mixing_matrix_.shape[1]
+    data = np.dot(
+        ica.mixing_matrix_[:, : ica.n_components_].T,
+        ica.pca_components_[: ica.n_components_],
+    )
+    # Create an empty array of size (n_components, 64, 64) to fit topo values
+    topo_array = np.zeros((n_components, 64, 64))
+
+    # f, ax = plt.subplots(1, ica.n_components_) #For Visualization
+    for j in range(n_components):
+        topo_ = np.flipud(topographic_map(data[j, :], ica.info))
+
+        # ax[j].imshow(topo_) # For visualization
+
+        # Set NaN values to 0
+        np.nan_to_num(topo_, copy=False)
+
+        # Standardize the values
+        topo_array[j, :, :] = topo_ / np.max(np.abs(topo_))
+
+    return topo_array  # topographic map array for all the components (n_components, 64, 64)
+
+
+def topographic_map(
+    data,
+    pos: Info,
+    vmin=None,
+    vmax=None,
+    res: int = 64,
+    outlines="head",
+    image_interp=_INTERPOLATION_DEFAULT,  #'cubic
+    border=_BORDER_DEFAULT,  #'mean'
+    extrapolate=_EXTRAPOLATE_DEFAULT,  #'head'
+):
+    """Generate a topographic map as image.
```

Comment:
```suggestion
    """Generate a scalp topographic map (n_pixels, n_pixels).
```

I'm mostly removing the words 'image' and 'plot' since they relate to visuals while we are just interested in the underlying data array.